app/services/attempts_api/tracker_events.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module AttemptsApi
+  module TrackerEvents
+    # @param [String] email The submitted email address
+    # @param [Boolean] success True if the email and password matched
+    # A user has submitted an email address and password for authentication
+    def email_and_password_auth(email:, success:)
+      track_event(
+        :email_and_password_auth,
```

Comment:
WDYT of using the dash name here so it can be easier to map to the OpenAPI docs w/o `.dasherize`?

Also WDYT of bringing back some of the YARD checking we used to have? (or maybe leave that for follow-up PR)

Reply:
Yep, makes sense

Also brought back the YARD checking in [031b693](https://github.com/18F/identity-idp/pull/11716/commits/031b6939f7a988ed71d47b628fe67a7c8cb3c26e)